ext/puma_http11/extconf.rb

```diff
@@ -14,6 +14,8 @@
   found_ssl = if (!$mingw || RUBY_VERSION >= '2.4') && (t = pkg_config 'openssl')
     puts 'using OpenSSL pkgconfig (openssl.pc)'
     true
+  elsif have_library('libcrypto', 'BIO_read') && have_library('libssl', 'SSL_CTX_new')
+    true
```

Comment:
Is this change related to this PR?

Reply:
This started as a PR about the mswin build, and it might be a bug in mkmf.  The RuboCop ENV variable involved the same code, so I added it.  An issue or PR mentioned the idea of having a way to skip RuboCop, but I couldn't find it.

But, almost all current OpenSSL implementations name the files 'libcrypto' and 'libssl'.  Older installs did not.  Older Windows OpenSSL used name 'libeay32' and 'ssleay32'.  Anyway, it is needed for mswin, and most other builds use pkgconfig.